docker-compose.yaml

```diff
@@ -1,5 +1,72 @@
----
-version: '3.0'
+version: '3.8'
+
+volumes:
```

Comment:
I like this `volumes` section. Did you do any testing with it to confirm bind mounts still work as expected?

Reply:
it's now been tested, and it works after making sure all mountpoints exist